### PR TITLE
Docs: issue 1961: fix qdstat procedures

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/master.adoc [hunk 1]
```diff
@@ -84,8 +84,12 @@ include::assemblies/assembly-upgrading-enmasse-kube.adoc[leveloffset=+3]
 === Monitoring {ProductName}
 
 include::assemblies/assembly-monitoring-oc.adoc[leveloffset=+3]
+
 include::assemblies/assembly-monitoring-kube.adoc[leveloffset=+3]
-include::assemblies/assembly-using-qdstat.adoc[leveloffset=+3]
+
+include::assemblies/assembly-using-qdstat-oc.adoc[leveloffset=+3]
```
Comment (ncbaratta) [inline documentation/master.adoc]: We're going to want to try and avoid having heading nested this deeply.

File: documentation/modules/proc-view-connections-qdstat.adoc [hunk 2]
```diff
@@ -9,11 +9,18 @@ You can view the router connections using `qdstat`.
 
 .Procedure
 
-* On the command line, run the following command:
+. On the command line, run the following command to obtain the `podname` value needed in the following step:
++
+[options="nowrap"]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-connections-qdstat.adoc]: You need the subs attributes thingy

File: documentation/modules/proc-view-connections-qdstat.adoc [hunk 3]
```diff
@@ -9,11 +9,18 @@ You can view the router connections using `qdstat`.
...
+----
+{cmdcli} get pods
+----
 
+. On the command line, run the following command:
++
 [source=bash,options="nowrap"]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-connections-qdstat.adoc]: You need the subs attributes thingy

File: documentation/modules/proc-view-link-routes-qdstat.adoc [hunk 4]
```diff
@@ -9,11 +9,18 @@ You can view the link routes using `qdstat`.
 
 .Procedure
 
-* On the command line, run the following command:
+. On the command line, run the following command to obtain the `podname` value needed in the following step:
++
+[options="nowrap"]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-link-routes-qdstat.adoc]: You need the subs attributes thingy

File: documentation/modules/proc-view-link-routes-qdstat.adoc [hunk 5]
```diff
@@ -9,11 +9,18 @@ You can view the link routes using `qdstat`.
...
+----
+{cmdcli} get pods
+----
 
+. On the command line, run the following command:
++
 [source=bash,options="nowrap"]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-link-routes-qdstat.adoc]: You need the subs attributes thingy

File: documentation/modules/proc-view-router-addresses-qdstat.adoc [hunk 6]
```diff
@@ -9,11 +9,18 @@ You can view the router addresses using `qdstat`.
 
 .Procedure
 
-* On the command line, run the following command:
+. On the command line, run the following command to obtain the `podname` value needed in the following step:
++
+[options="nowrap"]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-router-addresses-qdstat.adoc]: You need the subs attributes thingy

File: documentation/modules/proc-view-router-addresses-qdstat.adoc [hunk 7]
```diff
@@ -9,11 +9,18 @@ You can view the router addresses using `qdstat`.
...
+----
+{cmdcli} get pods
+----
 
-[source=bash,options="nowrap"]
+. Run the following command:
++
+[source=bash,options="nowrap",subs=quotes]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-router-addresses-qdstat.adoc]: You need the subs attributes thingy

File: documentation/modules/proc-view-router-links-qdstat.adoc [hunk 8]
```diff
@@ -9,11 +9,18 @@ You can view the router links using `qdstat`.
 
 .Procedure
 
-* On the command line, run the following command:
+. On the command line, run the following command to obtain the `podname` value needed in the following step:
++
+[options="nowrap"]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-router-links-qdstat.adoc]: You need the subs attributes thingy

File: documentation/modules/proc-view-router-links-qdstat.adoc [hunk 9]
```diff
@@ -9,11 +9,18 @@ You can view the router links using `qdstat`.
...
+----
+{cmdcli} get pods
+----
 
+. On the command line, run the following command:
++
 [source=bash,options="nowrap"]
```
Comment (ncbaratta) [inline documentation/modules/proc-view-router-links-qdstat.adoc]: You need the subs attributes thingy

File: documentation/assemblies/assembly-using-qdstat-kube.adoc [hunk 10]
```diff
@@ -0,0 +1,23 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+:parent-context: {context}
+:context: kube
+:cmdcli: kube
```
Comment (ncbaratta) [inline documentation/assemblies/assembly-using-qdstat-kube.adoc]: shouldn't this be 'cmdcli'

Comment (lulf) [reply documentation/assemblies/assembly-using-qdstat-kube.adoc]: Yes, it should be kubectl

**Comment by jenmalloy** [comment]
@ncbaratta @lulf, review comments addressed in latest commit

File: documentation/assemblies/assembly-using-qdstat-kube.adoc [hunk 11]
```diff
@@ -0,0 +1,23 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+:parent-context: {context}
+:context: kube
+:cmdcli: kubectl
+
+[id='using-qdstat-{context}']
+= Using qdstat on Kubernetes
```
Comment (ncbaratta) [inline documentation/assemblies/assembly-using-qdstat-kube.adoc]: Did you want to use the attribute for Kubernetes?

File: documentation/modules/proc-view-router-addresses-qdstat.adoc (hunk 6 above)
Comment (ncbaratta) [reply documentation/modules/proc-view-router-addresses-qdstat.adoc]: This is still an issue

**Pull request merged**